File: lib/backend/Symbol.dart
class Symbol{
  static int _counter=0;
   int _id;
  String pathToSymbolPic;//path for picture
  String name;

  Symbol(this.pathToSymbolPic,this.name){
    _id=++_counter;
  }

  int get id{
    return _id;
  }
}

File: functions/engine/Symbol.js
class Symbol{
  constructor(pathToSymbolPic, name) {
    this.counter = 0;
    this.id = ++this.counter
    this.name = name
    this.pathToSymbolPic = pathToSymbolPic
  }

  get id() {
    return this.id;
  }
}